Library/Formula/asciinema2gif.rb

```diff
@@ -0,0 +1,20 @@
+class Asciinema2gif < Formula
+  desc "Generate animated GIFs from asciinema terminal recordings"
+  homepage "https://github.com/tav/asciinema2gif"
+  url "https://github.com/tav/asciinema2gif/archive/0.2.tar.gz"
+  sha256 "fa5834aaec162b992768bed854d69eecff0862c733d4be0334dbc3b9fe86481c"
+  head "https://github.com/tav/asciinema2gif.git"
+
+  depends_on "gifsicle"
+  depends_on "imagemagick"
+  depends_on "phantomjs"
+
+  def install
+    bin.install "asciinema2gif"
+    bin.install "render.js"
```

Comment:
I have a question here, though. `render.js` needs to be in the same directory as `asciinema2gif`, * but* it does need to be symlinked, as it should never be called directly (it’s called by `asciinema2gif`). Is there a standard equivalent way to `bin.install` to move to the `bin` directory without symlinkg?